### PR TITLE
Header section of portfolio mobile version

### PR DESCRIPTION
**Pull request by Abass-rashid**
In this PR I have created the first two sections of the portfolio project 
1. header section with logo and hamburger menu
2. header line with introduction information
3.  covert header-footer in to a link tag
4.  added the background image
5. updated the README file

### CODE REVIEW

**Comment by Abass-rashid** [comment]
please have a review on it thanks

**Review by aimemalaika — CHANGES_REQUESTED** [review]
Hi @Abass-rashid, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- No linter errors. ✔️
- GitHub flow is followed. ✔️
- PR has a good title and summary. ✔️

 ## Required Changes ♻️
- [ ] Kindly update your readme following the microverse readme temple that you can find [here](https://github.com/microverseinc/readme-template)
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -2,14 +2,13 @@
 
 # Project Name
 
-> Description the project.
+> Portfolio project.
```
Comment (ShadyShawkat) [inline README.md:5]: - [ ] Please edit this section to contain the project's name and description.

File: README.md [hunk 2]
```diff
@@ -2,14 +2,13 @@
...
```
Comment (ShadyShawkat) [inline README.md:7]: - [ ] Kindly make sure to edit each section in your readme with the appropriate information to better suit your project and make it more professional. This also means that if there are sections that you don't have information for yet, then they should be removed. (Like the second author for the "Authors" section, "Run tests" section, "Deployment" section, ...etc)

**Review by m05e5 — APPROVED** [review]
# Hi @Abass-rashid , 

### Status: Approved ✔️
goodjob


:+1: You have done all the required changes asked by the previous reviewer. Well Done.

Your project is complete! 
Congratulations! 🎉 


**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_m05e5_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**